Makefile.gen.go.mk

```diff
@@ -119,6 +119,11 @@ lint: ## Runs golangci-lint.
 	@echo "====> $@"
 	golangci-lint run -E gosec -E goconst --timeout=15m ./...
 
+.PHONY: nancy
```

Comment:
Shouldn't this be done in the template file instead? https://github.com/giantswarm/devctl/blob/master/pkg/gen/input/makefile/internal/file/Makefile.gen.go.mk.template

Reply:
It is done there as well: https://github.com/giantswarm/devctl/pull/439/files#diff-ce91819b452e89cc8bc32e20acd6906351e52a25f03a34392886d23c68a7efc1

Eventually it would have generated it for itself via `github` repository.

Reply:
I kind of added here as well to test the change 😅